src/ValueObjects/RequestDuration.php

```diff
@@ -6,16 +6,22 @@
 
 class RequestDuration
 {
-    public function __construct(public ?int $milliseconds)
```

Comment:
Why did you delete the `__construct()`?

Reply:
Similar to the model, Laravel tries to inject the `RequestDuration` class in the event, but as it requires a param in the construct, Laravel doesn't know how to initialize that param. The good thing is that we always use the `from` method. So, in this case it's fine to remove the construct.